docs/GettingStartedDocs/VisualStudioWindows.md

```diff
@@ -0,0 +1,99 @@
+# Using Visual Studio to Develop Enclave Applications for Windows
+
+This walkthrough assumes that you are using Visual Studio on a Windows development
+machine, and want to develop enclaves for Windows.  If you instead want to use Visual
+Studio Code, see the
+[VS Code instructions](https://github.com/openenclave/openenclave/blob/master/devex/vscode-extension/README.md).
+
+## Prerequisites
+
+You will need the following:
+
+- A development machine with support for SGX1 or SGX1 with Flexible Launch Control (FLC) (see [here](https://github.com/microsoft/openenclave/blob/master/docs/GettingStartedDocs/SGXSupportLevel.md) for how to
```

Comment:
Should we provide instructions to set up a system that does not support SGX? In case a developer works on an enclave locally on a machine without SGX to publish it to ACC.

Reply:
Will look into which DLLs must be copied into the system from the PSW to have simulation mode work, even though the PSW won't install on a system that is not SGX-capable.

Reply:
Have the list; will go into a future PR.